source/manual/deployment_dashboards.html.md

```diff
@@ -0,0 +1,187 @@
+---
+owner_slack: '#2ndline'
+review_by: 2017-05-01
+title: Deployment Dashboards
```

Comment:
Maybe the title can be more action-y, something like "Monitor your app during deploy"?

Also see the new guide to a good title: https://docs.publishing.service.gov.uk/manual/create-page.html#title